### PR TITLE
Initalize project

### PR DESCRIPTION
**Pull request by rafalgolubowicz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "@typescript-eslint/eslint-plugin": "^5.12.1",
+    "@typescript-eslint/parser": "^5.12.1",
+    "eslint": "^8.9.0",
+    "eslint-config-prettier": "^8.4.0",
+    "eslint-config-standard": "^16.0.3",
+    "eslint-plugin-import": "^2.25.4",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^6.0.0",
+    "prettier": "^2.5.1",
+    "typescript": "^4.5.5"
```
Comment (Danizy) [inline package.json:27]: should not most of these, if not all be `devDependencies`?

File: packages/backend/.env.example [hunk 2]
```diff
@@ -0,0 +1,3 @@
+PORT=8080
+ORIGIN=""
+DATABASE_URL=""
```
Comment (Danizy) [inline packages/backend/.env.example]: please add newline at end of the file

File: packages/frontend/package.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    "@testing-library/user-event": "^13.5.0",
+    "@types/jest": "^27.4.0",
+    "@types/node": "^16.11.25",
+    "@types/react": "^17.0.39",
+    "@types/react-dom": "^17.0.11",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-scripts": "5.0.0",
+    "typescript": "^4.5.5",
+    "web-vitals": "^2.1.4"
```
Comment (Danizy) [inline packages/frontend/package.json]: please move types, testing libraries to `devDependencies`

File: packages/backend/src/utils/normalizePort.ts [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+  if (port >= 0) {
+    // port number
+    return port;
+  }
+
+  return false;
+};
+
+export { normalizePort };
```
Comment (Danizy) [inline packages/backend/src/utils/normalizePort.ts]: this could be default export

**Comment by Danizy** [comment]
There is `yarn.lock` in root project and `package.lock` in  `packages/frontend`. Please stick to one package manager

File: package.json (hunk 1 above)
Comment (rafalgolubowicz) [reply package.json:27]: indeed!

File: packages/backend/.env.example (hunk 2 above)
Comment (rafalgolubowicz) [reply packages/backend/.env.example]: indeed

**Pull request merged**